code/datums/uplink/medical.dm

```diff
@@ -6,36 +6,36 @@
 
 /datum/uplink_item/item/medical/sinpockets
 	name = "Box of Sin-Pockets"
-	desc = "A box of filled dough pockets. Great for a quick meal when you're hiding from Security. Instructions included on the box."
+	desc = "A box of filled dough pockets. Great for a quick meal. Instructions included on the box."
```

Comment:
Really should change this as the donk pocket inside it is meant to be a unique traitor item for healing

Reply:
The sinpockets inside don't contain any special chemical, it's just food.